src/blocks.ts

```diff
@@ -2,15 +2,15 @@ import { getDataForTable } from "./xl/data";
 
 const colour = "#107C41";
 
-export type Block = any;
-export type DataTable = any;
+// export type Block = any;
+// export type DataTable = any;
 
-export let currentWorkspace;
+export let currentWorkspace: unknown;
 
 export const blocks = [
     {
         kind: "block",
-        type: "excel_import_table",
+        type: "excelImportTable",
```

Comment:
don't rename these identifiers, they are used in the serialized blocky and will break existing files

Reply:
Added a note.